package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "bedrock-web-mock-kms-http",
+  "version": "0.0.1-0",
+  "description": "Mock Bedrock KMS service",
+  "main": "index.js",
+  "peerDependencies": {
+    "pretender": "^2.1.1"
```

Comment:
I gather that this whole module is somehow tied in with pretender, but I notice that `pretender` is not imported anywhere here, so I'm confused.

Reply:
Unfortunately, there can only be one pretender instance, so we have to instantiate it elsewhere and pass it in. Pretender's design is too restrictive; it did not consider modular, reusable mock backends.